oggsound/include/OgreOggSoundManager.h

```diff
@@ -926,6 +928,7 @@ namespace OgreOggSound
 		ALCchar* mDeviceStrings;				// List of available devices strings
 		unsigned int mNumSources;				// Number of sources available for sounds
 		unsigned int mMaxSources;				// Maximum Number of sources to allocate
+		ALuint* mSources;
```

Comment:
```suggestion
		std::vector<ALuint> mSources;
```

Reply:
@paroj this is not a good idea, because `alGenSources()` takes a `ALuint*` as second argument. Same thing with `alDeleteSources()`

Reply:
you can pass `mSources.data()` there